### DISCUSSION TITLE
Mastodon account for this project

### DISCUSSION THREAD

**Discussion by linux-lukas** [grote/Transportr#789] [General]
Would a Mastodon account for this project be possible?

**Comment by ialokim** [comment]
I think it would be nice to be present on Mastodon as well, but we hardly have time to work on the app itself and therefore we unfortunately won't be able to spend more time for social media purposes :/

**Reply by linux-lukas** [reply]
I would basically offer my help to organize a Mastodon account in parts.

**Reply by linux-lukas** [reply]
For this, it should be clarified which instance and whether @grote is also OK with it.

**Reply by grote** [reply]
Maybe https://fosstodon.org mostly mirroring the Twitter account?

**Reply by linux-lukas** [reply]
Fosstodon.org could be considered.

Twitter crosspost could be done, even if it is often not seen quite so favourably in the Fediverse community.

**Reply by Altonss** [reply]
I'm willing to open an account for Transportr on the fediverse on fosstodon, if you want!
But I don't really know if crossposting is a good idea, I would rather post exclusive content, maybe on the new developement efforts :)

**Reply by Echolon** [reply]
> I'm willing to open an account for Transportr on the fediverse on fosstodon, if you want! But I don't really know if crossposting is a good idea, I would rather post exclusive content, maybe on the new developement efforts :)

Yes, and maybe we should just keep it as remote to the Transportr Twitter account for the moment. And yes, in general I recommend to focus on the app and don't expand to much outside of the app related content.
